task3/index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+  <meta charset="utf-8">
+  <head>
+    <title>Sales For You</title>
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@500&family=Pattaya&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header>
+        <div>
+            <img src="images/Profil.png" class="imgProfil" alt="profil">
+            <p class="nik">y.hodoss</p>
+        </div>
+        <h1>Sales For You</h1>
+    </header>
+
+    <main>
+            <details>
+                <summary>Фильтровать</summary>
+                <button>По дате</button>
+                <button>По хештегам</button>
+                <button>По поставщику</button>
+            </details>
+            <table width="100%" cellspacing="0" cellpadding="5">
+            <tr> 
+            <td width="730px" valign="top">
+                <fieldset>
```

Comment:
`<fieldset>` tag is used to group several controls as well as labels (<label>) within a web form (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset). Please use some other tag here.